samples/automl/automlNaturalLanguageModel.js

```diff
@@ -0,0 +1,599 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on dataset
+ * with the Google AutoML Natural Language API.
+ *
+ * For more information, see the documentation at
+ * https://cloud.google.com/natural-language/automl/docs/
+ */
+
+`use strict`;
+
+/* eslint node/no-extraneous-require: off */
+
+function createModel(projectId, computeRegion, datasetId, modelName) {
+  // [START automl_natural_language_createModel]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset`;
+  // const modelName = `Name of the model, e.g. "myModel"`;
+
+  // A resource that represents Google Cloud Platform location.
+  const projectLocation = client.locationPath(projectId, computeRegion);
+
+  // Set model name and model metadata for the dataset.
+  const myModel = {
+    displayName: modelName,
+    datasetId: datasetId,
+    textClassificationModelMetadata: {},
+  };
+
+  // Create a model with the model metadata in the region.
+  client
+    .createModel({parent: projectLocation, model: myModel})
+    .then(responses => {
+      const operation = responses[0];
+      const initialApiResponse = responses[1];
+
+      console.log(`Training operation name: ${initialApiResponse.name}`);
+      console.log(`Training started...`);
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      const model = responses[0];
+
+      // Retrieve deployment state.
+      var deploymentState = ``;

```

Comment:
Use `let`, not `var`. Apply this comment throughout your PR(s).

Also, ideally this would use a ternary operator or an array - e.g:

```javascript
const stateLabels = [null, "deployed, undeployed"];
const deploymentState = stateLabels[model.deploymentState];
```